src/atomate2/forcefields/md.py

```diff
@@ -0,0 +1,377 @@
+    from ASE (`MolecularDynamics`), the user can specify the dynamics as a string or an
+    ASE class into the `dynamics` attribute. In this case, please consult the ASE
+    documentation for the parameters and units to pass into the ASE MD function through
+    `ase_md_kwargs`.
+
+    Parameters
+    ----------
+    name : str
+        The name of the MD Maker
+    force_field_name : str
+        The name of the forcefield (for provenance)
+    timestep : float | None = 2.
+        The timestep of the MD run in femtoseconds
+    nsteps : int = 1000
+        The number of MD steps to run
+    ensemble : str = "nvt"
+        The ensemble to use. Valid ensembles are nve, nvt, or npt
+    temperature: float | Sequence | np.ndarray | None.
+        The temperature in Kelvin. If a sequence or 1D array, the temperature
+        schedule will be interpolated linearly between the given values. If a
+        float, the temperature will be constant throughout the run.
+    pressure: float | Sequence | None = None
+        The pressure in kilobar. If a sequence or 1D array, the pressure
+        schedule will be interpolated linearly between the given values. If a
+        float, the pressure will be constant throughout the run.
+    dynamics : str | ASE .MolecularDynamics = "langevin"
+        The dynamical thermostat to use. If dynamics is an ASE .MolecularDynamics
+        object, this uses the option specified explicitly by the user.
+        See _valid_dynamics for a list of pre-defined options when
+        specifying dynamics as a string.
+    ase_md_kwargs : dict | None = None
+        Options except for temperature and pressure to pass into the ASE MD function
+    calculator_kwargs : dict
+        kwargs to pass to the ASE calculator class
+    traj_file : str | Path | None = None
+        If a str or Path, the name of the file to save the MD trajectory to.
+        If None, the trajectory is not written to disk
+    traj_file_fmt : Literal["ase","pmg"]
+        The format of the trajectory file to write. If "ase", writes an
+        ASE trajectory, if "pmg", writes a Pymatgen trajectory.
+    traj_interval : int
+        The step interval for saving the trajectories.
+    mb_velocity_seed : int | None = None
+        If an int, a random number seed for generating initial velocities
+        from a Maxwell-Boltzmann distribution.
+    zero_linear_momentum : bool = False
+        Whether to initialize the atomic velocities with zero linear momentum
+    zero_angular_momentum : bool = False
+        Whether to initialize the atomic velocities with zero angular momentum
+    task_document_kwargs: dict
+        Options to pass to the TaskDoc. Default choice
+        {
+            "store_trajectory": "partial",
+            "ionic_step_data": ("energy",),
+        }
+        is consistent with atomate2.vasp.md.MDMaker
+    """
+
+    name: str = "Forcefield MD"
+    force_field_name: str = f"{MLFF.Forcefield}"
+    timestep: float | None = 2.0
```

Comment:
Added both requested changes. If a structure contains an isotope of H, `time_step = 0.5` fs, and 2 fs otherwise. Also changed the behavior so that it isn't assumed that a user specifies `time_step` in fs